lnbits/extensions/subdomains/cloudflare.py

```diff
@@ -20,25 +18,21 @@ async def cloudflare_create_subdomain(
         "Content-Type": "application/json",
     }
     aRecord = subdomain + "." + domain.domain
```

Comment:
- the call in [tasks.py](https://github.com/lnbits/lnbits/blob/8bf410352331e46bf99a7cebc002159db668756e/lnbits/extensions/subdomains/tasks.py#L32) does not hand raised exceptions from `cloudflare_create_subdomain`

Reply:
added try / except in tasks.py